package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curriculumassociates/createjs-accessibility",
-  "version": "0.3.0",
+  "version": " 0.3.1",
```

Comment:
Nit: extra space at the start of the version string.  I'll take care of this in a separate PR to not hold this one up any longer.